torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2239,12 +2239,10 @@ options :class:`~torch.distributed.ProcessGroupNCCL.Options`).
               py::arg("timeout") = ::c10d::kProcessGroupNCCLDefaultTimeout,
               py::call_guard<py::gil_scoped_release>())
           .def(
-              "_abort",
-              [](const c10::intrusive_ptr<::c10d::ProcessGroupNCCL>& self,
-                 const c10::optional<std::string>& abortReason) {
-                return self->abort(abortReason);
+              "_shutdown",
+              [](const c10::intrusive_ptr<::c10d::ProcessGroupNCCL>& self) {
+                return self->shutdown();
```

Comment:
Is this supporting gloo too or shall we add the `_shutdown` for gloo to keep consistency?

Reply:
This really depends if we want `shutdown` as a public method in the `ProcessGroup.hpp` public interface and have all subclasses implement it (not just gloo).

I kept this private here since I wasn't sure if we want this to be part of the ProcessGroup interface.